sandboxes/viper.py

```diff
@@ -0,0 +1,19 @@
+import io
+import logging
+import requests
+from common import parse_config
+conf = parse_config()
+
+logger = logging.getLogger('pastehunter')
+
+def upload_file(raw_file, paste_object):
+    viper_ip = conf["sandboxes"]["viper"]["api_host"]
+    viper_port = conf["sandboxes"]["viper"]["api_port"]
+    viper_host = 'http://{0}:{1}'.format(viper_ip, viper_port)
```

Comment:
Same here, would like http/https to be configurable

Reply:
Same comment as https://github.com/kevthehermit/PasteHunter/pull/87#discussion_r329375900